chore(client): migrate i18n/configForTest.js to TS

alcaidet opened the pull request:
Checklist:

<!-- Please follow this checklist and put an x in each of the boxes, like this: [x]. It will ensure that our team takes your pull request seriously. -->

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

<!--If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.-->

Is part of [#43778](https://github.com/freeCodeCamp/freeCodeCamp/issues/43778)

<!-- Feel free to add any additional description of changes below this line -->
Hello, I am a first time contributor. I sincerely appreciate any feedback so please let me know if there are any improvements I should make. Thank you so much in advance for all of your time and help!

alcaidet committed:
fix: change file extension to .ts

alcaidet committed:
fix(client): migrate configForTests.js to ts

raisedadead reviewed (changes requested):
Can we do kebab-case for the filename? You would have to update wherever this is used as well.

alcaidet committed:
fix(client): rename configForTests.ts to config-for-tests.ts

alcaidet commented:
> Can we do kebab-case for the filename? You would have to update wherever this is used as well.

Thanks so much for catching that! I apologize for the oversight. I've refactored the file to kebab-case format and updated the `app-mount-notifier-test.tsx` file which uses it. 

Please let me know if there is anything else. Thanks again!

raisedadead commented:
#### :sparkles: :sparkles: **AWESOME!** :sparkles: :sparkles: 

LGTM.

Thanks for this pull request, @alcaidet, and for contributing to the code-base for the first time.

We will be looking forward to more contributions in the future.

Cheers and happy contributing!
